api/metrics/src/main/java/io/opentelemetry/api/metrics/MeterProvider.java

```diff
@@ -27,7 +27,9 @@
    * @param instrumentationName The name of the instrumentation library, not the name of the
    *     instrument*ed* library.
    * @return a meter instance.
+   * @deprecated Use {@link #meterBuilder(String)}
    */
+  @Deprecated
   default Meter get(String instrumentationName) {
```

Comment:
I think we might want to keep this one around, since it'll be easy mode for application developers who don't care about version/schemaUrl. Thoughts?

Reply:
If we do I'd like to add that helper to the log sdk as well.

@anuraaga do you have thoughts on this? Was your comment in the log sdk PR related to removing `get(String instrumentationName)` and `get(String instrumentationName, String instrumentationVersion, String schemaUrl)`?

Reply:
> If we do I'd like to add that helper to the log sdk as well.

Logs SDK will only be used by the appender instrumentation, not directly by the user, won't it?
The `MeterProvider` needs to be called whenever you want to add a custom counter to your app - and usually the instrumentation name is already confusing enough even without the builder and the other params.

Reply:
I wonder if we should call it "loggerName" or some such in the logging SDK, as that will be more familiar to users of logging APIs.

Reply:
> Logs SDK will only be used by the appender instrumentation, not directly by the user, won't it?
Because Otel Logs are covering both conventional application logs [as well as events](https://github.com/open-telemetry/opentelemetry-specification/issues/2074#issuecomment-954781513), I can definitely see a strong argument that the SDK _would_ be used directly in applications. 

> I wonder if we should call it "loggerName" or some such in the logging SDK, as that will be more familiar to users of logging APIs.
The spec currently doesn't talk about instrumentation library, and how its involved in mapping logs from logging frameworks. Not sure if that's covered under one of the outstanding issues, eg [Issue #2069](https://github.com/open-telemetry/opentelemetry-specification/issues/2069).

Reply:
Sorry I did mean to keep one get method with just the library name, I probably missed the removal from the logs PR. It's nice to have the quick, but not parameter-exploding, factory method along a builder.